### PR TITLE
 Issue 212 To add next steps in success message

### PR DESCRIPTION
**Pull request by DhanashriPalodkar**
**What type of PR is this?**

/kind cleanup

**What does this PR do / why we need it**:
clarifies success message after the execution of the bootstrap command

**Have you updated the necessary documentation?**
Yes

See https://github.com/rhd-gitops-example/docs

[ ] Updated

**Which issue(s) this PR fixes**:
https://issues.redhat.com/browse/GITOPS-212

Fixes #?
212

### CODE REVIEW

File: pkg/odo/cli/pipelines/bootstrap.go [hunk 1]
```diff
@@ -75,7 +75,16 @@ func (io *BootstrapParameters) Run() error {
 	if err != nil {
 		return err
 	}
-	log.Success("Bootstrapped GitOps sucessfully.")
```
Comment (bigkevmcd) [inline pkg/odo/cli/pipelines/bootstrap.go:109]: We probably only need to link to one URL.

Also, this should not be duplicated, let's extract it to a separate function?

Comment (DhanashriPalodkar) [reply pkg/odo/cli/pipelines/bootstrap.go:109]: sure, I guess boottrap command will be completely removed so just to test the output I kept it there. Are we keeping both bootstrap and wizard command?

File: pkg/odo/cli/pipelines/bootstrap.go [hunk 2]
```diff
@@ -75,7 +75,16 @@ func (io *BootstrapParameters) Run() error {
...
+	log.Success("Bootstrapped Openshift resources (GitOps) sucessfully. \n",
+		"Next Steps\n",
+		"1. Bring the bootstrapped environment up. \n",
```
Comment (bigkevmcd) [inline pkg/odo/cli/pipelines/bootstrap.go]: I think we should shorten to this to something like...

"Next Steps\n",
"Deploying your new Pipelines configuration https://github.com/rhd-gitops-example/docs/tree/master/journey/day1#bringing-the-bootstrapped-environment-up"

Comment (bigkevmcd) [reply pkg/odo/cli/pipelines/bootstrap.go]: The numbered steps don't have links to say what those things mean.

**Review by bigkevmcd — CHANGES_REQUESTED** [review]

**Pull request merged**